packages/google_identity_services_web/example/integration_test/utils.dart

```diff
@@ -2,31 +2,47 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// the following ignore is needed for downgraded analyzer (casts to JSObject).
+// ignore_for_file: unnecessary_cast
+
 import 'dart:async';
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
 
+import 'package:flutter_test/flutter_test.dart';
+import 'package:google_identity_services_web/id.dart';
 import 'package:google_identity_services_web/oauth2.dart';
-import 'package:google_identity_services_web/src/js_interop/dom.dart';
-import 'package:js/js.dart';
-import 'package:js/js_util.dart';
+import 'package:web/web.dart' as web;
+
+/// Function that lets us expect that a JSObject has a [name] property that matches [matcher].
+///
+/// Use [createExpectConfigValue] to create one of this functions associated with
+/// a specific [JSObject].
+typedef ExpectConfigValueFn = void Function(String name, Object? matcher);
 
-@JS('window')
-external Object get domWindow;
+/// Creates a [ExpectConfigValueFn] for the `config` [JSObject].
+ExpectConfigValueFn createExpectConfigValue(JSObject config) {
+  return (String name, Object? matcher) {
+    expect(config.getProperty(name.toJS), matcher, reason: name);
```

Comment:
Alternatively: `config[name]` here and elsewhere where `getProperty` is used with a `String` :)

Reply:
Oh my, of course I couldn't find any `.get`, never tried to look at the `operator []` 😅